code/src/java/pcgen/io/Compatibility.java

```diff
@@ -83,8 +82,7 @@ static PCTemplate getTemplateFor(PCTemplate template,
 		return null;
 	}
 
-	@NotNull
-	public static String getKeyFor(PCTemplate pct)
+	public static @NotNull String getKeyFor(PCTemplate pct)
```

Comment:
Are these equivalent?  I thought outside meant it applied to parameters and return?